downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc

```diff
@@ -0,0 +1,27 @@
+//this module appears in assembly-collections-and-content-signing-in-pah
+
+[id="proc-downloading-signature-public-keys"]
+
+= Downloading signature public keys
+
+After you sign and approve collections, download the signature public keys from the {HubName} UI. You must download the public key before you add it to the local system keyring.
```

Comment:
Hope this doesn't seem too picky, but I would only use the imperative type sentences (e.g. "download the signature public keys..." in the procedures. Here's just a suggestion. Of course, it's up to you to decide if this works. 

```suggestion
After you sign and approve collections, you must download the signature public keys from the {HubName} UI before you add it to the local system keyring.
```

Reply:
Thank you, @jself-sudoku! Please review and squash and merge if you are satisfied w/the changes.